openmc/deplete/abc.py

```diff
@@ -547,7 +547,11 @@ class Integrator(ABC):
         :attr:`solver`.
 
         .. versionadded:: 0.12
+    transfer_rates : openmc.deplete.TransferRates
+        Instance of TransferRates class to perform continuous transfer based
+        on transfer rates definitions.
 
+        .. versionadded:: 0.13.3
```

Comment:
Sorry this keeps getting bumped up! :laughing: 
```suggestion
        .. versionadded:: 0.13.4
```